docs/eq_snapshot.md

```diff
@@ -1,7 +1,7 @@
 ## General
 
 A snapshot can be compared against any value with `==`.
-The value gets recorded if the snapshot is undefined (`snapshot()`)
+The value can be recorded with `--inline-snapshot=create` if the snapshot is empty.
```

Comment:
FYI, as described in https://github.com/15r10nk/inline-snapshot/issues/57#issuecomment-2018930088, I use `--inline-snapshot=fix` very liberally so that I very rarely have to type it. This means I also never use `create`, I just type `snapshot(0)` (because `0` is easiest to type inside `()`) and then `fix` the `0`. I don't know what the advantage is of requiring `create` for `snapshot()`, why not just make `fix/update` work there as well?

Reply:
the idea is that create will only create new snapshots which are currently empty and **not change existing one**.

This allows to add new `snapshot()` and create the empty one without having to fear to accidentally changing existing ones.

This is only an option for the user. You can also use `--inline-snapshot=create,fix` to do both at the same time.

I will try to explain this better when I redo some parts of the documentation.

Reply:
But `fix` doesn't work for empty snapshots, right?

Reply:
No, this is the reason why flags can be combined like `--inline-snapshot=create,fix`.
Every change is assigned to one flag. And the flags allow you to select which changes you want to apply.

Reply:
What's the advantage of `fix` not working with `snapshot()`? Why force the user to use `create`?

Reply:
there are currently 4 flags which have no overlapping (create,fix,trim,update)

https://15r10nk.github.io/inline-snapshot/pytest/

(I think the flags should be called categories, this would fit better)

The thing is that you could ask the same question for update. should fix imply update?

This is actually sometime the case in the current version and it is not easy to explain when fix will also update other snapshots. This should be gone with the new implementation.

I wanted to divide this reasons for snapshot changes to give the user the control above what he wants to change.

Long story short, I could make fix work for `snapshot()` but this would mix these categories/flags together and the benefit would be that the user does not have to type some characters (which are probably in already in your history). I wanted to separate these categories and chose the current implementation.